scripts/test_firmware_prepare.py

```diff
@@ -54,4 +54,4 @@ def test_layout(tmpdir: pathlib.Path) -> None:
         with tempfile.TemporaryDirectory() as temp_unzip_dir:
             shutil.unpack_archive(file, temp_unzip_dir)
 
-            assert len(list(pathlib.Path(temp_unzip_dir).glob("*"))) == 7
+            assert len(list(pathlib.Path(temp_unzip_dir).glob("*"))) == 12
```

Comment:
Can you make this 12 a const at the top of the file (well pythongs fake const, all caps)